### PR TITLE
Safety label input device permission

### LINKED ISSUE #2249: Safety label input device permission
Show a safety label for https://github.com/flatpak/flatpak/pull/5481

### PR DESCRIPTION
**Pull request by razzeee**
Closes #2249

### CODE REVIEW

File: frontend/src/safety.ts [hunk 1]
```diff
@@ -163,6 +164,18 @@ export function getSafetyRating(
       icon: HiOutlineVideoCamera,
       showOnSummaryOrDetails: "both",
     })
+  } else if (
+    summary.metadata.permissions.devices?.some(
+      (x) => x.toLowerCase() === "input",
```
Comment (hfiguiere) [inline frontend/src/safety.ts:169]: there is no `--device=input` for flatpak. Not sure what this is about.

File: frontend/public/locales/en/common.json [hunk 2]
```diff
@@ -404,6 +404,7 @@
   "uses-system-services": "Uses system services",
   "uses-session-services": "Uses session services",
   "can-access-hardware-devices": "Can access hardware devices such as webcams or gaming controllers",
+  "can-access-input-devices": "Can access input devices such as keyboards or mice or gaming controllers",
```
Comment (TingPing) [inline frontend/public/locales/en/common.json:407]:
```suggestion
  "can-access-input-devices": "Can access input devices such as keyboards, mice, or gaming controllers",
```

File: frontend/src/safety.ts (hunk 1 above)
Comment (TingPing) [reply frontend/src/safety.ts:169]: I swore I had seen it in the wild before, but yeah it doesn't exist.

Comment (razzeee) [reply frontend/src/safety.ts:169]: I don't think it has reached any app yet - might still take some time for compatibility, but it does exist? https://github.com/flatpak/flatpak/pull/5481

Comment (TingPing) [reply frontend/src/safety.ts:169]: Ah it is very new, nevermind.

**Pull request merged**